accelerometer/summariseEpoch.py

```diff
@@ -7,13 +7,16 @@
 import pandas as pd
 import pytz
 import sys
-
+import scipy as sp
+from scipy import fftpack
+from datetime import timedelta
 
 def getActivitySummary(epochFile, nonWearFile, summary,
     activityClassification=True, startTime=None, endTime=None,
     epochPeriod=30, stationaryStd=13, minNonWearDuration=60, mgMVPA=100,
     mgVPA=425, activityModel="activityModels/doherty2018.tar",
-    intensityDistribution=False, verbose=False):
+    intensityDistribution=False, psd=False, fourierFrequency=False, m10l5=False, 
+    verbose=False, fourierWithAcc=False):
```

Comment:
fourierWithAcc should immediately follow fourierFrequency in the list of arguments here ... make sure to update in accProcess.py to reflect this (and anywhere else it might be called)

Reply:
resolved in bbe4536c65b673e2a6d7ec09277fc17e132c6837